aspnetcore/mvc/views/view-components.md

```diff
@@ -107,6 +107,12 @@ Pascal-cased class and method parameters for Tag Helpers are translated into the
 </vc:[view-component-name]>
 ```
 
+Note that in order to use a View Component as a Tag Helper, you must register the assembly containing the View Component using the `@addTagHelper` directive. For example, if your View Component is in an assembly called "MyWebApp" you will need the following directive in your `_ViewImports.cshtml` file to register the View Components:
```

Comment:
I don't know if I'd state that it must exist in your `_ViewImports.cshtml` file. A user could put the directive anywhere they want.